docs/source/checks/tabular/model_evaluation/plot_confusion_matrix_report.py

```diff
@@ -4,6 +4,20 @@
 
 Confusion Matrix Report
 ***********************
+This notebook provides an overview for using and understanding the Confusion Matrix Report check.
+
+
+**Structure:**
+
+* `What is the Confusion Matrix Report? <#what-is-the-confusion-matrix-report>`__
+* `Generate data & model <#generate-the-data>`__
```

Comment:
link won't work since no relevant title
```suggestion
* `Generate data & model <#generate-data-model>`__
```

fixing link according to title update